docs/app/Examples/views/Feed/Types/Props.js

```diff
@@ -38,11 +38,11 @@ const events = [
   {
     date: '4 days ago',
     image: 'http://semantic-ui.com/images/avatar/small/justen.jpg',
-    meta: <Feed.Like>41 Likes</Feed.Like>,
-    summary: <div> <a>Justen Kitsune</a> added <a>2 new photos</a> of you</div>,
+    meta: '41 Likes',
+    summary: 'Justen Kitsune added 2 new photos of you',
     extraImages: [
-      <a><img src='http://semantic-ui.com/images/wireframe/image.png' /></a>,
-      <a><img src='http://semantic-ui.com/images/wireframe/image.png' /></a>,
+      'http://semantic-ui.com/images/wireframe/image.png',
+      'http://semantic-ui.com/images/wireframe/image.png',
```

Comment:
Even though we support Icon/Image components elsewhere, I'm considering only supporting strings for simplicity and clarity in v1.  Will leave these examples as strings even though our prop renderer supports both.